test/e2e/placement_apply_strategy_test.go

```diff
@@ -77,11 +78,21 @@ var _ = Describe("validating CRP when resources exists", Ordered, func() {
 		})
 
 		AfterAll(func() {
+			By("deleting created work resources on member cluster")
```

Comment:
Hi Britania! The `AfterAll` block might run in case of test interruptions/failures between steps -> in that case it would make sense to drop the CRP first then deleting the resources to avoid unexpected inconsistencies.